WooCommerce/Classes/ViewRelated/Dashboard/Settings/Settings/SettingsViewModel.swift

```diff
@@ -238,7 +238,10 @@ private extension SettingsViewModel {
         let storeSettingsSection: Section? = {
             var rows: [Row] = []
 
-            if stores.sessionManager.defaultSite?.isJetpackCPConnected == true {
+            let site = stores.sessionManager.defaultSite
+            if site?.isJetpackCPConnected == true ||
```

Comment:
Same as the other comment: 

Super nit: Can `site?.isJetpackCPConnected == true` be shortened as just `site?.isJetpackCPConnected`?

Reply:
Same as above, we cannot check for optional `bool` without unwrapping or comparing it to `true`.